### ISSUE TITLE
SyntaxError Missing ")" Preventing Any Use on Raspberry Pi

### ISSUE THREAD

**Issue by creedda** [dhleong/ps4-waker#45]
I had PS4-Waker working fine on my rpi3 for a couple of months. Recently however ps4-waker has stopped working. I have tried updating it, and reinstalling it with no success. Here is the error I get when I run ps4-waker with any arguments (-v, -h, nothing, it doesn't matter)

Running Node version 6.10.3. Do I need to update to 7.0.0?

I have removed my credentials in my home directory in case that was part of the problem. So I would expect it to take me though the setup process. Instead I get the error below:

```
pi@raspberrypi:~ $ ps4-waker -h
/usr/lib/node_modules/ps4-waker/bin/cmd.js:100
    action = doAndClose(async function(device) {
                        ^^^^^
SyntaxError: missing ) after argument list
    at createScript (vm.js:56:10)
    at Object.runInThisContext (vm.js:97:10)
    at Module._compile (module.js:542:28)
    at Object.Module._extensions..js (module.js:579:10)
    at Module.load (module.js:487:32)
    at tryModuleLoad (module.js:446:12)
    at Function.Module._load (module.js:438:3)
    at Module.runMain (module.js:604:10)
    at run (bootstrap_node.js:390:7)
    at startup (bootstrap_node.js:150:9)
pi@raspberrypi:~ $ npm -v
4.6.1
pi@raspberrypi:~ $ node -v
v6.10.3
```

Let me know if there is any other information I can gather to help you troubleshoot, but I am hoping we can fix it.

Thanks,

**Comment by dhleong** [comment]
Ack, that's my fault. Updating to 7 should fix it, but I'll release an update shortly. Forgot that `cmd.js` isn't being transpiled for node6 and I can't use async functions there

**Comment by creedda** [comment]
No problem. Glad I could help 👍 

If you are going to update I'll just wait for that. When it starts working again I will close this as fixed. If you need anything from me let me know.

If you feel like updating to 7 would be a better option in the long run for more functionality, let me know that too. Just trying to keep on the most stable version.

**Comment by dhleong** [comment]
Released 1.3.3; please give it a shot and let me know if there're any other problems for you.

I'd love to *just* support 7, but since Node6 is still LTS until like October, I feel I should support it.

**Comment by creedda** [comment]
Works like a charm. Thanks for this awesome app!
